packages/auth/cloud/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
You created a package (`@ttoss/cloud`) inside another package (`@ttoss/auth`). It doesn't work.